apis/v1alpha1/generated.proto

```diff
@@ -602,12 +602,13 @@ message HTTPRouteMatch {
   // PathType is defines the semantics of the `Path` matcher.
   //
   // Support: core (Exact, Prefix)
-  // Support: extended (RegularExpression)
-  // Support: custom (ImplementationSpecific)
+  // Support: custom (RegularExpression, ImplementationSpecific)
   //
   // Default: "Exact"
   //
   // +optional
+  // +kubebuilder:validation:Enum=Exact;Prefix;RegularExpression;ImplementationSpecific
+  // +kubebuilder:default=Exact
```

Comment:
Do these work in the correct order? I know other things like required annotations do not work with default values.

Reply:
I hope this works, we use it in `HostnameMatch` and `GatewayAddress` too.

Reply:
I applied the CRD and the basic example and it correctly populated the default. I'm using k8s 1.16.9.